// @opd/g2plot-react/lib/index.js
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault").default;

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "BaseChart", {
  enumerable: true,
  get: function () {
    return _base.default;
  }
});
Object.defineProperty(exports, "AreaChart", {
  enumerable: true,
  get: function () {
    return _area.default;
  }
});
Object.defineProperty(exports, "BarChart", {
  enumerable: true,
  get: function () {
    return _bar.default;
  }
});
Object.defineProperty(exports, "BulletChart", {
  enumerable: true,
  get: function () {
    return _bullet.default;
  }
});
Object.defineProperty(exports, "ColumnChart", {
  enumerable: true,
  get: function () {
    return _column.default;
  }
});
Object.defineProperty(exports, "FunnelChart", {
  enumerable: true,
  get: function () {
    return _funnel.default;
  }
});
Object.defineProperty(exports, "GaugeChart", {
  enumerable: true,
  get: function () {
    return _gauge.default;
  }
});
Object.defineProperty(exports, "HeatmapChart", {
  enumerable: true,
  get: function () {
    return _heatmap.default;
  }
});
Object.defineProperty(exports, "HistogramChart", {
  enumerable: true,
  get: function () {
    return _histogram.default;
  }
});
Object.defineProperty(exports, "LineChart", {
  enumerable: true,
  get: function () {
    return _line.default;
  }
});
Object.defineProperty(exports, "LiquidChart", {
  enumerable: true,
  get: function () {
    return _liquid.default;
  }
});
Object.defineProperty(exports, "PieChart", {
  enumerable: true,
  get: function () {
    return _pie.default;
  }
});
Object.defineProperty(exports, "RadarChart", {
  enumerable: true,
  get: function () {
    return _radar.default;
  }
});
Object.defineProperty(exports, "RoseChart", {
  enumerable: true,
  get: function () {
    return _rose.default;
  }
});
Object.defineProperty(exports, "ScatterChart", {
  enumerable: true,
  get: function () {
    return _scatter.default;
  }
});
Object.defineProperty(exports, "WordCloudChart", {
  enumerable: true,
  get: function () {
    return _wordCloud.default;
  }
});
Object.defineProperty(exports, "ProgressChart", {
  enumerable: true,
  get: function () {
    return _progress.default;
  }
});
Object.defineProperty(exports, "RingProgressChart", {
  enumerable: true,
  get: function () {
    return _ringProgress.default;
  }
});
Object.defineProperty(exports, "TinyAreaChart", {
  enumerable: true,
  get: function () {
    return _tinyArea.default;
  }
});
Object.defineProperty(exports, "TinyColumnChart", {
  enumerable: true,
  get: function () {
    return _tinyColumn.default;
  }
});
Object.defineProperty(exports, "TinyLineChart", {
  enumerable: true,
  get: function () {
    return _tinyLine.default;
  }
});
Object.defineProperty(exports, "WaterfallChart", {
  enumerable: true,
  get: function () {
    return _waterfall.default;
  }
});
Object.defineProperty(exports, "DualAxesChart", {
  enumerable: true,
  get: function () {
    return _dualAxes.default;
  }
});
Object.defineProperty(exports, "BoxChart", {
  enumerable: true,
  get: function () {
    return _box.default;
  }
});
Object.defineProperty(exports, "StockChart", {
  enumerable: true,
  get: function () {
    return _stock.default;
  }
});
Object.defineProperty(exports, "SunburstChart", {
  enumerable: true,
  get: function () {
    return _sunburst.default;
  }
});
Object.defineProperty(exports, "RadialBarChart", {
  enumerable: true,
  get: function () {
    return _radialBar.default;
  }
});
Object.defineProperty(exports, "BidirectionalBarChart", {
  enumerable: true,
  get: function () {
    return _bidirectionalBar.default;
  }
});
Object.defineProperty(exports, "SankeyChart", {
  enumerable: true,
  get: function () {
    return _sankey.default;
  }
});
Object.defineProperty(exports, "TreemapChart", {
  enumerable: true,
  get: function () {
    return _treemap.default;
  }
});
Object.defineProperty(exports, "ChordChart", {
  enumerable: true,
  get: function () {
    return _chord.default;
  }
});
Object.defineProperty(exports, "MultiViewChart", {
  enumerable: true,
  get: function () {
    return _multiView.default;
  }
});
Object.defineProperty(exports, "MixChart", {
  enumerable: true,
  get: function () {
    return _mix.default;
  }
});

var _base = _interopRequireDefault(require("./components/base"));

var _area = _interopRequireDefault(require("./plots/area"));

var _bar = _interopRequireDefault(require("./plots/bar"));

var _bullet = _interopRequireDefault(require("./plots/bullet"));

var _column = _interopRequireDefault(require("./plots/column"));

var _funnel = _interopRequireDefault(require("./plots/funnel"));

var _gauge = _interopRequireDefault(require("./plots/gauge"));

var _heatmap = _interopRequireDefault(require("./plots/heatmap"));

var _histogram = _interopRequireDefault(require("./plots/histogram"));

var _line = _interopRequireDefault(require("./plots/line"));

var _liquid = _interopRequireDefault(require("./plots/liquid"));

var _pie = _interopRequireDefault(require("./plots/pie"));

var _radar = _interopRequireDefault(require("./plots/radar"));

var _rose = _interopRequireDefault(require("./plots/rose"));

var _scatter = _interopRequireDefault(require("./plots/scatter"));

var _wordCloud = _interopRequireDefault(require("./plots/word-cloud"));

var _progress = _interopRequireDefault(require("./plots/progress"));

var _ringProgress = _interopRequireDefault(require("./plots/ring-progress"));

var _tinyArea = _interopRequireDefault(require("./plots/tiny-area"));

var _tinyColumn = _interopRequireDefault(require("./plots/tiny-column"));

var _tinyLine = _interopRequireDefault(require("./plots/tiny-line"));

var _waterfall = _interopRequireDefault(require("./plots/waterfall"));

var _dualAxes = _interopRequireDefault(require("./plots/dual-axes"));

var _box = _interopRequireDefault(require("./plots/box"));

var _stock = _interopRequireDefault(require("./plots/stock"));

var _sunburst = _interopRequireDefault(require("./plots/sunburst"));

var _radialBar = _interopRequireDefault(require("./plots/radial-bar"));

var _bidirectionalBar = _interopRequireDefault(require("./plots/bidirectional-bar"));

var _sankey = _interopRequireDefault(require("./plots/sankey"));

var _treemap = _interopRequireDefault(require("./plots/treemap"));

var _chord = _interopRequireDefault(require("./plots/chord"));

var _multiView = _interopRequireDefault(require("./plots/multi-view"));

var _mix = _interopRequireDefault(require("./plots/mix"));
